### PR TITLE
Poc stale element exception

### PR DESCRIPTION
**Pull request by vladb00mer**

### CODE REVIEW

**Comment by codacy-review** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=4817150&bid=16018710)

File: jdi-light-html-tests/src/test/java/io/github/epam/html/tests/elements/common/ButtonTests.java [hunk 1]
```diff
@@ -6,8 +6,8 @@
 import org.testng.annotations.Test;
 
 import static com.epam.jdi.light.common.Exceptions.safeException;
-import static com.epam.jdi.light.elements.common.Alerts.acceptAlert;
-import static com.epam.jdi.light.elements.common.Alerts.getAlertText;
+import static com.epam.jdi.light.elements.common.Alerts.*;
```
Comment (codacy-review) [inline jdi-light-html-tests/src/test/java/io/github/epam/html/tests/elements/common/ButtonTests.java]: Codacy Issue found: [Avoid unused imports such as 'com.epam.jdi.light.elements.common.Alerts'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=4817150)

File: jdi-light/src/main/java/com/epam/jdi/light/elements/base/JDIBase.java [hunk 2]
```diff
@@ -444,18 +445,14 @@ private static String printWebElement(WebElement element) {
...
-    }
-    private Actions actions = null;
-    private Actions actionsClass() {
-        if (actions == null)
-            actions = new Actions(driver());
-        return actions;
+    public void actionsWitElement(JFunc2<Actions, WebElement, Actions> action) {
+        action.execute(actions.get().moveToElement(get()), get()).build().perform();
     }
+    private Safe<Actions> actions = new Safe<>(() -> new Actions(driver()));
```
Comment (codacy-review) [inline jdi-light/src/main/java/com/epam/jdi/light/elements/base/JDIBase.java]: Codacy Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=4817150)

**Comment by RomanIovlev** [comment]
fixed

**Pull request closed without merging**